entity-services/src/main/xdmp/entity-services/entity-services-impl.xqy

```diff
@@ -656,10 +656,11 @@ declare function esi:database-properties-generate(
         for $word-lexicon-property in json:array-values($word-lexicon-properties)
         let $property := $entity-type=>map:get("properties")=>map:get($word-lexicon-property)
         let $collation := head( (map:get($property, "collation"), "http://marklogic.com/collation/en") )
+        let $namespace-uri := head( ($entity-type=>map:get("namespace"), "") )
         let $wl-map := json:object()
             =>map:with("collation", $collation)
             =>map:with("localname", $word-lexicon-property)
-            =>map:with("namespace-uri", "")
+            =>map:with("namespace-uri", $namespace-uri)
```

Comment:
We now get $namespace-uri for "namespace-uri" field if the model doesnt have any namespaces.

 ```
 "element-word-lexicon":[  
    {  
      "collation":"http://marklogic.com/collation/",
      "localname":"orderId",
      "namespace-uri":"$namespace-uri"
    }
```

You can see this in unit test testDatabasePropertiesGenerate() 
`fn:doc('SchemaCompleteEntityType-0.0.1.json')=>es:database-properties-generate()`

Otherwise this PR looks good.